src/ng/directive/select.js

```diff
@@ -76,9 +76,11 @@ var SelectController =
     }
   };
 
+  var scopeDestroyed = false;
```

Comment:
Yes, I could. But since we were already having a `$destroy` callback I opted for using that instead of the `$$destroy` as it seems unnecssary. Can go either way.